### ISSUE TITLE
seed should not be global

### ISSUE THREAD

**Issue by totty90** [josephg/noisejs#10]
you should instance a Noise and set the seed on that one.

**Comment by Hypercubed** [comment]
I agree with this... the module should really return a constructor so the seed can be set once per instance.  Something like:

```
var myNoise = Noise(seed);
var z = myNoise.perlin2(x,y);
```

**Comment by BenMcLean** [comment]
Yeah, making this global is pretty ridiculous. There is no reason it should be global.

**Comment by as-f** [comment]
I tried to do this: #19 
This is my first time doing anything like this, so I'm not sure if I'm going about it in the right way, but I replaced the anonymous function with a named one and gave it the seed as a parameter.
